src/index.ts

```diff
@@ -11,6 +11,7 @@ export default async function init(options?: Options) {
   const nearConnection = await connect({
     keyStore,
     ...nearConfig,
+    //@ts-ignore
```

Comment:
We should remove this.

Reply:
On my machine it won't build without this ts-ignore